app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<% @recipes.each do |recipe| %>
+  <div class="row">
+    <div>
+      <h2> Recipe <%= recipe.id %> </h2>
+      <%= link_to "Delete", user_recipe_path(user_id: current_user.id, id: recipe.id), class: "btn btn-outline-danger", "data-turbo-method": :delete %>
```

Comment:
- [ ] Kindly move the delete button below the description. It is not proper to have it in between the name and description.